app/scripts/models/auth_brokers/fx-ios-v1.js

```diff
@@ -11,13 +11,88 @@ define(function (require, exports, module) {
   'use strict';
 
   const _ = require('underscore');
-  const FxDesktopV1AuthenticationBroker = require('../auth_brokers/fx-desktop-v1');
+  const FxDesktopChannel = require('../../lib/channels/fx-desktop-v1');
+  const FxSyncChannelAuthenticationBroker = require('../auth_brokers/fx-sync-channel');
+  const HaltBehavior = require('../../views/behaviors/halt');
   const NavigateBehavior = require('../../views/behaviors/navigate');
+  const Url = require('../../lib/url');
   const UserAgent = require('../../lib/user-agent');
 
-  const proto = FxDesktopV1AuthenticationBroker.prototype;
+  const proto = FxSyncChannelAuthenticationBroker.prototype;
+
+  const FxiOSV1AuthenticationBroker = FxSyncChannelAuthenticationBroker.extend({
+    type: 'fx-desktop-v1',
+
+    commands: {
+      CAN_LINK_ACCOUNT: 'can_link_account',
+      CHANGE_PASSWORD: 'change_password',
+      DELETE_ACCOUNT: 'delete_account',
+      LOADED: 'loaded',
+      LOGIN: 'login'
+    },
+
+    defaultBehaviors: _.extend({}, proto.defaultBehaviors, {
+      // about:accounts displays its own screen after sign in, no need
+      // to show anything.
+      afterForceAuth: new HaltBehavior(),
+      // about:accounts displays its own screen after password reset, no
+      // need to show anything.
+      afterResetPasswordConfirmationPoll: new HaltBehavior(),
+      // about:accounts displays its own screen after sign in, no need
+      // to show anything.
+      afterSignIn: new HaltBehavior(),
+      // about:accounts displays its own screen after sign in, no need
+      // to show anything.
+      afterSignInConfirmationPoll: new HaltBehavior(),
+      // about:accounts displays its own screen after sign in, no need
+      // to show anything.
+      afterSignUpConfirmationPoll: new HaltBehavior()
+    }),
+
+    createChannel () {
+      var channel = new FxDesktopChannel();
+
+      channel.initialize({
+        // Fx Desktop browser will send messages with an origin of the string
+        // `null`. These messages are trusted by the channel by default.
+        //
+        // 1) Fx on iOS and functional tests will send messages from the
+        // content server itself. Accept messages from the content
+        // server to handle these cases.
+        // 2) Fx 18 (& FxOS 1.*) do not support location.origin. Build the origin from location.href
```

Comment:
We no longer support either of these environments, it's a good opportunity to remove outdated logic (and comment!).

On line 63, can you remove from `||` to the end of the line?